xml/System.Runtime.Intrinsics/Vector128.xml

```diff
@@ -89,11 +90,12 @@
         <Parameter Name="vector" Type="System.Runtime.Intrinsics.Vector128&lt;T&gt;" RefType="this" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="vector">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of the input vector.</typeparam>
+        <param name="vector">The vector to reinterpret.</param>
+        <summary>Reinterprets a &lt;see cref="T:System.Runtime.Intrinsics.Vector128`1" /&gt; as a new &lt;see cref="T:System.Runtime.Intrinsics.Vector128`1" /&gt;.</summary>
```

Comment:
It doesn't seem that it likes the &lt; and &gt; characters:
https://review.docs.microsoft.com/en-us/dotnet/api/system.runtime.intrinsics.vector128.asbyte?view=netcore-3.0&branch=pr-en-us-2278

![image](https://user-images.githubusercontent.com/12971179/55845258-04c53b00-5af6-11e9-972d-a208f44d76de.png)